user-guide/Advanced_Modules/DOM/tutorials/DOM_Getting_Started_With_DOM.md

```diff
@@ -0,0 +1,270 @@
+---
+uid: DOM_Getting_Started_With_DOM
+---
+# Getting started with DOM
+
+This tutorial shows how you can create a simple DOM model using the DOM Editor and then use it in a low-code app.
+
+The content and screenshots for this tutorial were created in DataMiner version 10.4.1.
+
+Expected duration: 20 minutes
+
+> [!TIP]
+> See also: [Kata #15: Getting started with DataMiner Object Models](https://community.dataminer.services/courses/kata-15/) on DataMiner Dojo ![Video](~/user-guide/images/video_Duo.png)
+
+## Prerequisites
+
+- A DataMiner System using DataMiner 10.3.10 or higher, which is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+- An [indexing database](xref:Indexing_Database) or [Storage as a Service](xref:STaaS).
+
+> [!NOTE]
+> If you use a [DaaS system](xref:Creating_a_DMS_in_the_cloud), these prerequisites are automatically met.
+
+## Overview
+
+- [Getting started with DOM](#getting-started-with-dom)
+  - [Prerequisites](#prerequisites)
+  - [Overview](#overview)
+  - [Step 1: Deploy the DOM Editor script](#step-1-deploy-the-dom-editor-script)
+  - [Step 2: Create a DOM definition](#step-2-create-a-dom-definition)
+  - [Step 3: Use the DOM definition in a low-code app](#step-3-use-the-dom-definition-in-a-low-code-app)
+
+## Step 1: Deploy the DOM Editor script
+
+1. Go to <https://catalog.dataminer.services/details/automation-script/3195>.
+
+1. Click the *Deploy* button to deploy the DOM Editor script to your DMA.
+
+## Step 2: Create a DOM definition
+
+Now that you have deployed the DOM Editor to your DMA, you can use it to create a DOM definition:
+
+1. [Open DataMiner Cube](xref:Opening_the_desktop_app) and go to *Apps* > *Automation*.
+
+1. In the Automation module, search for a script called **DOM Editor** and select it.
+
+   ![Dom Editor script](~/user-guide/images/DOM_Getting_Started_script.png)
+
+1. Click the *Execute* button in the lower right corner to launch this script, and then click *Execute now*.
+
+   You should now see the following window:
+
+   ![Dom Editor](~/user-guide/images/DOM_Getting_Started_with_DOM_DOMEditor.png)
+
+1. Click *New* to create a new DOM module.
+
+1. In the *Module ID* box, enter `eventmanagement`.
+
+   This is the name of the module that you are creating.
+
+   ![Dom Editor New Module](~/user-guide/images/DOM_Getting_Started_with_DOM_DOMModule.png)
+
+1. Click *Create*.
+
+   This should result a new module in the list of available DOM modules:
+
+   ![Dom Editor Module created](~/user-guide/images/DOM_Getting_Started_with_DOM_DOMModuleCreated.png)
```

Comment:
Could you remove the mouse pointer from this screenshot? As a rule, unless you're actually demonstrating what the user should click, it's better not to show the pointer (to avoid possible confusion).

Reply:
Done